### PR TITLE
add submission link to intro

### PR DESCRIPTION
**Pull request by kmartens27**
This is a suggestion to add a link to the user story submission site, since the form is no longer being used. I tried to utilize the same link structure used elsewhere in the page and ran it through a linter to make sure that there weren't any errors introduced. 

This is also my first attempt at adding a link in .jsx so if I used the incorrect syntax, I can definitely update it to what it should be.

### CODE REVIEW

File: src/pages/all.jsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const AllPage = () => {
                         <h2>Tell Your Story</h2>
                         <p>&quot;Jenkins Is The Way&quot; is a global showcase of how developers and engineers are building, deploying, and automating great stuff with Jenkins.</p>
                         <p>Share your story and we'll send you a free Jenkins Is The Way T-shirt.</p>
-                        <p>Submit a pull request about your project's goals and technical challenges, and the unique solutions you encountered with Jenkins.</p>
+                        <p><Link to={{ pathname: 'https://stories.jenkins.io/admin/#/collections/user-story' }} target="_blank">Share the story of your project's goals, technical challenges, and the unique solutions you encountered with Jenkins.</Link></p>
```
Comment (halkeye) [inline src/pages/all.jsx:43]: Link is only used for inside the gastby app (it does some fancy react magic so it doesn't need to download everything again).
```suggestion
                        <p><a href="/admin/#/collections/user-story" target="_blank">Share the story of your project's goals, technical challenges, and the unique solutions you encountered with Jenkins.</a></p>
```

I'm not a fan of target=_blank, I think users should be allowed to decide when a new page is opened and when its not. When its forced, you can't unforce it.

Comment (kmartens27) [reply src/pages/all.jsx]: Agree, thanks very much @halkeye!

**Review by halkeye — APPROVED** [review]
i still don't like target, but lgtm

**Comment by kmartens27** [comment]
@halkeye, I went back and removed the target to avoid the new tab

**Pull request merged**